### PR TITLE
[Backend][Improvement] Using Generated Configurations files

### PR DESCRIPTION
**Pull request by mohitdeuex**
### Describe your changes :

We are using hardcoded generated files,
OMD being a Schema first application, json schema can be used to generate models for configurations as well

#
### Type of change :

- [x] Improvement
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: catalog-rest-service/src/main/resources/json/schema/configuration/elasticSearchConfiguration.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "port": {
+      "description": "Elastic Search port",
+      "type": "integer"
+    },
+    "scheme": {
+      "description": "Http/Https connection scheme",
+      "type": "string"
+    },
+    "username": {
+      "description": "Elastic Search Host",
```
Comment (vivekratnavel) [inline catalog-rest-service/src/main/resources/json/schema/configuration/elasticSearchConfiguration.json:22]: We need to update all the descriptions.

File: catalog-rest-service/src/main/resources/json/schema/configuration/jwtTokenConfiguration.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/configuration/jwtTokenConfiguration.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AuthenticationConfiguration",
```
Comment (vivekratnavel) [inline catalog-rest-service/src/main/resources/json/schema/configuration/jwtTokenConfiguration.json:4]:
```suggestion
  "title": "JWT Token Configuration",
```

File: catalog-rest-service/src/main/resources/json/schema/configuration/slackEventPubConfiguration.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/configuration/slackEventPubConfiguration.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AuthenticationConfiguration",
```
Comment (vivekratnavel) [inline catalog-rest-service/src/main/resources/json/schema/configuration/slackEventPubConfiguration.json:4]: Need to update the title and description

File: catalog-rest-service/src/main/resources/json/schema/configuration/elasticSearchConfiguration.json (hunk 1 above)
Comment (mohitdeuex) [reply catalog-rest-service/src/main/resources/json/schema/configuration/elasticSearchConfiguration.json]: done

**Review by vivekratnavel — APPROVED** [review]

File: catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplicationConfig.java [hunk 4]
```diff
@@ -36,82 +36,98 @@
...
+  private static final String EVENTHANDLER_CONFIG = "eventHandlerConfiguration";
+  private static final String AIRFLOW_CONFIG = "airflowConfiguration";
+  private static final String SLACK_CONFIG = "slackEventPublishers";
+  private static final String FERNET_CONFIG = "fernetConfiguration";
+  private static final String HEALTH_CONFIG = "health";
+  private static final String MIGRATION_CONFIG = "migrationConfiguration";
+  private static final String SANDBOXMODE_CONFIG = "sandboxModeEnabled";
+  private static final String SLACK_CHAT_CONFIG = "slackChat";
+  private static final String SECRET_MANAGER_CONFIG = "secretsManagerConfiguration";
+
```
Comment (sureshms) [inline catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplicationConfig.java:54]: @mohitdeuex and @vivekratnavel, no need to define private static variables. We can just use the string as it is used only once and with in this class.

**Pull request merged**